website/content/docs/release-notes/consul-api-gateway/v0_4_x.mdx

```diff
@@ -0,0 +1,72 @@
+---
+layout: docs
+page_title: 0.4.x
+description: >-
+  Consul API Gateway release notes for version 0.4.x
+---
+
+# Consul API Gateway 0.4.0
+
+## Release Highlights
+
+- **Support for Kubernetes Gateway API Version 0.5.0 and v1beta1 APIs:**
+  The `v0.5.0` release of the Kubernetes Gateway API is significant because it
+  marks the growth in maturity to a beta API version (v1beta1) release for some
+  of the key APIs:
+  - GatewayClass
+  - Gateway
+  - HTTPRoute
+
+  The other APIs (e.g. TCPRoute) are still at the `v1alpha2` stage.
+
+  Reaching `v1beta1` status has several benefits for users, including greater
+  stability and backward compatibility requirements. Existing fields and allowed
+  options can not be removed or renamed except in a new, major version of the
+  API. Once an API reaches `v1beta1` status, future versions must comply with
+  several backward compatibility requirements.
+
+- **URL Path Prefix Rewrite**
+  This release introduces support for rewriting a URL's path prefix when routing
+  HTTP traffic. This is configured by adding a `URLRewrite` filter to a
+  `HTTPRoute`. With this feature, the gateway can rewrite the URL path, in a
+  client's HTTP Request, before sending the request to a service. A simple
+  example of this is changing the path from `//store/checkout` to
+  `//cart/checkout`. Please see the product documentation for details on how to
+  configure this feature.
```

Comment:
```suggestion
  HTTP traffic. To use this functionality, add a `URLRewrite` filter to an
  `HTTPRoute` configuration. This enables the gateway to rewrite the URL path in a client's HTTP request before sending the request to a service. For example, you could configure the gateway to change the path from `//store/checkout` to `//cart/checkout`. Refer to the [usage documentation](/docs/api-gateway/usage) for additional information.  
```
Mostly just fixing the commas and applying a few adjustments for compliance with the style guide